m4_sessions/1711-inning/Group_6.md

```diff
@@ -6,18 +6,38 @@ Description: Why should students come to this session?
   As technology professional we will have to deal with privacy issues of users we are responsible for providing services for.  There are times where you will have to make a moral decision instead of a legal one.  How do you go about deciding if a legal technical action is moral or not?  Should companies take a moral stand on anything related to the security of their users information?
 
 Session Outcomes: What will students leave with or gain from this session?
-  An appreciation for ethical issues involving security and user data and a consideration for circumstances that, while legal, may not be moral.  They will be forced to consider several hypothetical situations where they may be tempted to behave unethically.
+  An appreciation for ethical issues involving security and user data and a consideration for circumstances that, while legal, may not be moral.  They will be forced to consider several hypothetical situations where they have to consider the tradeoff between user security and providing private information for the public good.
 
 Session Outline: Include the flow of the session and the time for each section.
+
 10 minutes - Icebreaker/Introductions
-10 minutes - Introduction (either a reading or a video)
+    Lead off session with Adam's video of a squirrel barking to inspire the group
+    Make your favorite animal noise or just any animal noise they can think of
+    As a group, everyone make their favorite animal noise together
+
+10 minutes - Introduction What this video and/or read the article about this 
+   (https://www.cbsnews.com/news/apple-and-the-fbi-split-on-creating-a-backdoor-to-access-phone-data/)
+
 15 minutes - Break into groups (facilatated by student leads) and discuss a scenario
+   2 (or more groups)
+   Try and pull out for and against arguments from group members
+   Discuss various answers
+   
 30 minutes - Group discussion 
+   3 Scenarios with information about the cell phones
+   These will be similar to the trolley problem.  If there is information about an attack that will result in 5,000,000 
+   people dying, would it be okay to decrypt the phone then?  If the information would only result in an attack that kills 
+   one person, would it be okay then?
+   
 10 minutes - Wrapup
-
+  Wrap it all up.  User privacy is something we have to consider as developers.
+  Software engineers make decisions regarding software design.  It most likely won't be as dramatic as protecting people in
+  the US from terrorists, but take Netflix identifying a depressed individual for example.  A developer likely created the
+  functionality to get this information.  Should Netflix use this user's private to reach out to them about if they are
```

Comment:
I would use this scenario in the above conversation.  Don't hold it until the end.